addons/forum-user-agent/userscript.js

```diff
@@ -0,0 +1,9 @@
+export default async function ({ addon, console }) {
+  if (!window.mySettings) return;
+  const ua = window.mySettings.markupSet.find((x) => x.className);
+  ua.openWith = window._simple_http_agent = ua.openWith.replace("version", "versions");
+  if (location.pathname === "/discuss/3/topic/add/") {
+    const textarea = document.getElementById("id_body");
+    textarea.value = ua.openWith;
```

Comment:
Add `dynamicEnable:false` to the manifest explicitly so that we avoid data loss here (we don't want a future contributor to think it's a good idea to add dynamicEnable to this addon)

Reply:
This also may be racy (possibly! maybe some image takes too long to work). Maybe add `runAtComplete:false` and change this to a waitForElement

Reply:
Have there been any issues other forum addon because of this?

Reply:
No, but this is enabled by default, so we can be more careful